### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by nikon4uk**

### CODE REVIEW

**Review by Oleksandr563487953 — APPROVED** [review]

File: tests/test_main.py [hunk 1]
```diff
@@ -1 +1,17 @@
...
+        mock_exchange_rate_prediction.return_value = 11
+        assert cryptocurrency_action(10) == "Buy more cryptocurrency"
+
+    def test_call_to_sell_cryptocurrency(self, mock_exchange_rate_prediction):
+        mock_exchange_rate_prediction.return_value = 9
+        assert cryptocurrency_action(10) == "Sell all your cryptocurrency"
+
+    def test_call_to_hold_cryptocurrency(self, mock_exchange_rate_prediction):
+        mock_exchange_rate_prediction.return_value = 10
+        assert cryptocurrency_action(10) == "Do nothing"
```
Comment (MasterpieceElbow) [inline tests/test_main.py:17]: Add boundary conditions, you don't check when `get_exchange_rate_prediction` return value equals exactly 1.05 or 0.95 * value passed in `cryptocurrency_action`.

**Review by MasterpieceElbow — CHANGES_REQUESTED** [review]

**Review by IvanRamyk — APPROVED** [review]
Well done!
